### PR TITLE
[FIX] web: ignore numpad for hotkeys

### PR DESCRIPTION
**Pull request by adwid**
To reproduce the issue:
(Need a Windows machine)
1. Install Stock
2. Inventory > Products > Products
3. Try to write the diameter symbol (ALT + 155) in the search bar

Error: When pressing the key '1' on the keyboard, the page is redirected
to the Inventory Overview page

Because of hotkeys, it is not possible to use the Windows feature that
allows users to write ASCII characters with ALT+[numerical code from
keypad].

OPW-2731632

### CODE REVIEW

File: addons/web/static/tests/core/hotkeys/hotkey_service_tests.js [hunk 1]
```diff
@@ -672,3 +672,18 @@ QUnit.test("protects editable elements: can bypassEditableProtection", async (as
...
+    const key = '1';
+
+    env.services.hotkey.add(`alt+${key}`, () => assert.step(key));
+    await nextTick();
+
+    let keydown = new KeyboardEvent("keydown", { key, code: "Numpad1", altKey: true });
+    window.dispatchEvent(keydown);
+    await nextTick();
+
+    assert.verifySteps([]);
```
Comment (brboi) [inline addons/web/static/tests/core/hotkeys/hotkey_service_tests.js:688]: add another keydown event asserting that it works when event.code is something else than NumpadX, like DigitX (which is the code format for keys from the numeric row)

**Review by brboi — CHANGES_REQUESTED** [review]

File: addons/web/static/tests/core/hotkeys/hotkey_service_tests.js (hunk 1 above)
Comment (brboi) [reply addons/web/static/tests/core/hotkeys/hotkey_service_tests.js:688]: good job btw ;-)

**Review by brboi — APPROVED** [review]

**Comment by aab-odoo** [comment]
robodoo delegate=brboi

**Comment by brboi** [comment]
robodoo r+

**Pull request closed without merging**